bindata/network/host-network/resourcequota.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: host-network-namespace-quotas
+  namespace: openshift-host-network
+spec:
+  hard:
```

Comment:
You can also restrict pods here with
```yaml
pods: "0"
count/deployments.apps: "0"
count/daemonsets.apps: "0"
```

Reply:
isn't that redundant? Can someone create pods when memory and cpu allowed are `0`?

Reply:
I believe so; I believe you could create the pod, but scheduling will fail. Better to just block creation of the api object.

Reply:
Done.